src/Compilers/Core/Portable/Text/LargeText.cs

```diff
@@ -22,7 +21,7 @@ internal sealed class LargeText : SourceText
         /// <remarks>
         /// internal for unit testing
         /// </remarks>
-        internal const int ChunkSize = SourceText.LargeObjectHeapLimitInChars; // 40K Unicode chars is 80KB which is less than the large object heap limit.
+        internal const int ChunkSize = SourceText.LargeObjectHeapLimitInChars; // 40K
```

Comment:
I moved this comment to `SourceText.LargeObjectHeapLimitInChars`, where I think it's more appropriate.